docker/ansible-in-docker/build.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -vx
+
+function buildAnsible {
+    dockerTag=oso-ansible:${ansibleVersion}
+
+    bashScript=./oso-ansible-playbook-${ansibleVersion}
```

Comment:
I'm not sure RPM would work. With deprecating containers based on ansible versions it would need to match whatever containers are available. We could create these seperately to the build and do registry scanning and/or fixed (in repo) versions without scanning?